docs/coloring.md

```diff
@@ -37,7 +39,17 @@ To select a customize color:
     -  **Red**, **Green**, **Blue**: Enter the RGB Color. The triangle dial selector will move then to the selected RGB value.
     -  **Opacity**: Enter in the value desired (0 - 255). The lower the number chosen the more translucent the color will be.
     -  **Color Name**: If you know the color name you want you can directly input the value. Entering in a number will change the RGB color and move the dial selector settings to the corresponding color. If the RGB color or the triangle dial selector was used to select a color this color name value will be updated.  
-    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color.  An equivalent "Eye Dropper" type tool can also be accessed from the **Tools** menu (referenced as a Color Picker ![color picker](img/overview/picker.png)). 
+    -  **Eye Dropper**: Click on the eyedropper ![eye dropper](img/color/eyedropper.png) and then click a color anywhere on your screen to select that color. The corresponding color values will then be displayed on the **Choose Primary Color** or **Choose Secondary Color** dialog box.  
+
+## Select an Exact Color Match From an Image  
+    
+   1. Select the *Color Picker* tool ![color picker](img/overview/picker.png) from the **Tools** menu
+   
+      ![eye dropper](img/color/colorpicker.png).
+   
+   2. Left click to set the primary color and right click to set the secondary color.
+   3. Drag the *Color Picker* tool ![color picker](img/overview/picker.png) across the image and click on the image color pixel you would like to capture.

```

Comment:
3. Move the **Color Picker** ![color picker](img/overview/picker.png) across the workspace and click on the image color pixel you would like to capture.